upload.prx.org/upload.prx.org.yml

```diff
@@ -81,5 +111,24 @@ Resources:
       Handler: index.handler
       MemorySize: 128
       Role: !GetAtt LambdaBasicExecutionIAMRole.Arn
-      Runtime: nodejs4.3
+      Runtime: nodejs6.10
       Timeout: 5
+  UploadErrorAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: "${EnvironmentTypeAbbreviation} upload lambda errors"
+      AlarmActions:
+        - !Ref OpsWarnMessagesSNSTopicArn
```

Comment:
it is now (in this PR), I believe: https://github.com/PRX/Infrastructure/pull/30/files#diff-65f4f5695da65033aa4fcf7ce0414bfbL391